awx/ui_next/src/screens/Inventory/InventoryGroups/InventoryGroupDetail/InventoryGroupDetail.jsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from 'react';
+import { t } from '@lingui/macro';
+
+import { CardBody, Button } from '@patternfly/react-core';
+import { withI18n } from '@lingui/react';
+import { withRouter, Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { VariablesInput as CodeMirrorInput } from '@components/CodeMirrorInput';
+import ErrorDetail from '@components/ErrorDetail';
+import AlertModal from '@components/AlertModal';
+import { formatDateString } from '@util/dates';
+
+import { GroupsAPI } from '@api';
+import { DetailList, Detail } from '@components/DetailList';
+
+const VariablesInput = styled(CodeMirrorInput)`
+  .pf-c-form__label {
+    font-weight: 600;
+    font-size: 16px;
+  }
+  margin: 20px 0;
+`;
+const ActionButtonWrapper = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 20px;
+  & > :not(:first-child) {
+    margin-left: 20px;
+  }
+`;
+function InventoryGroupDetail({ i18n, history, match, inventoryGroup }) {
+  const {
+    summary_fields: { created_by, modified_by },
+  } = inventoryGroup;
+  const [error, setError] = useState(false);
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
+
+  const handleDelete = async () => {
+    setIsDeleteModalOpen(false);
+    try {
+      await GroupsAPI.destroy(inventoryGroup.id);
+      history.push(`/inventories/inventory/${match.params.id}/groups`);
+    } catch (err) {
+      setError(err);
+    }
+  };
+
+  return (
+    <CardBody style={{ paddingTop: '20px' }}>
+      <DetailList gutter="sm">
+        <Detail label={i18n._(t`Name`)} value={inventoryGroup.name} />
+        <Detail
+          label={i18n._(t`Description`)}
+          value={inventoryGroup.description}
+        />
+      </DetailList>
+      <VariablesInput
+        id="inventoryGroup-variables"
+        readOnly
+        value={inventoryGroup.variables}
+        rows={4}
+        label={i18n._(t`Variables`)}
+      />
+      <DetailList>
+        {created_by && created_by.username && (
+          <Detail
+            label={i18n._(t`Created`)}
+            value={
+              <span>
+                {i18n._(t`${formatDateString(inventoryGroup.created)} by`)}{' '}
+                <Link to={`/users/${created_by.id}`}>
+                  {created_by.username}
+                </Link>
+              </span>
+            }
+          />
+        )}
+        {modified_by && modified_by.username && (
```

Comment:
We should handle the case where `modified` is defined but `modified_by` is not.  This is how we do it for templates: https://github.com/ansible/awx/blob/devel/awx/ui_next/src/screens/Template/JobTemplateDetail/JobTemplateDetail.jsx#L115-L139